module/API/Controller.h

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file Controller.h
+ * @brief モーター制御に用いる関数をまとめたラッパークラス
+ * @author aridome222
+ */
+#ifndef CONTROLLER_H
+#define CONTROLLER_H
+
+#include "ev3api.h"
+#include "Motor.h"
+#include "Clock.h"
+
+class Controller {
+ public:
+  Controller();
+  Controller(const Controller&) = delete;  // 明示的にインスタンス化を禁止
```

Comment:
これ、インスタンスのコピーを禁止してるだけで、インスタンスが自体は禁止できてない気がする。

```suggestion
  Controller() = delete;  // 明示的にインスタンス化を禁止
```
まだ、試してないけど、こういうイメージだった。